redux-two/src/components/CreateToDo.js

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { addToDo } from '../actions/toDoAction';
+import {useDispatch} from 'react-redux'
+let nextId = 3;
+
+const CreateToDo = () => {
+    const [toDo, setToDo] = useState("");
+    const dispatch = useDispatch(); 
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        dispatch(addToDo({id: nextId++, toDo}))
```

Comment:
a best practice for this would be creating a variable and setting it to your new todo, then adding that variable to your addTodo action. this makes it more readable